### ISSUE TITLE
AS: accessing configuration from verifiers

### ISSUE THREAD

**Issue by thomas-fossati** [confidential-containers/trustee#384]
I would like to use the AS configuration file to describe the parameters for my verifier.

Something like:

```json
{
  "//": "global AS config"

  "work_dir": "...",

  "//": "per-verifier stanzas"
  "verifiers": {
    "cca": {
      "veraison": {
        "endpoint": "https://veraison.example/challenge-response/v1/newSession",
        "api-token": "/path/to/api/token.jwt"
        "tls-config": {
          "ca-certs": "/path/to/ca.pem",
          "//": "etc."
        }
      }
    }
  }
}
```

At present, this is not supported.  Is that a conscious design decision, or is it because no one had this need before?

A (less intrusive) alternative would be to have a separate configuration file and pass it to my verifier in an environment variable.

Would such an approach be preferable?

**Comment by Xynnn007** [comment]
@thomas-fossati Currently, the address for `verasion` is set by env key `VERAISON_ADDR` and we haven't supported the config. The reason might be no one had this need.

For a long term design, I prefer to have a pure rust CCA verifier handler/implementation to replace veraison for CCA.

Kevin (Hi @kevinzs2048 , hopefully I pinged the right person) could probably give more contexts.

**Comment by fitzthum** [comment]
Yeah I think I would prefer configuration through the config file over an environment variable.
